packages/server/test/support/fixtures/projects/e2e/cypress/integration/cookies_spec_baseurl.coffee

```diff
@@ -205,7 +205,8 @@ describe "cookies", ->
                 value: 'foo'
               })
 
-            cy.request('/requestCookies').its('body').should('include', { 'domaincookie': 'foo' })
+            cy.request("#{Cypress.config('baseUrl')}/requestCookies").its('body').should('include', { 'domaincookie': 'foo' })
```

Comment:
move the `"#{Cypress.config('baseUrl')}/requestCookies"` to a `const` and reuse in all the subsequent places